### ISSUE TITLE
Users should not be able to delete other users' posts

### ISSUE THREAD

**Issue by JacksonBates** [JacksonBates/pair-frontend#27]
At the moment the delete button naively allows anyone to delete any post.

Ideally, only the user that created the post should be allowed to delete a post.

In the longer term, we'll restructure the app so that users must be authenticated to allow use of the app (both posting and deleting).

A short term fix could be to use cookies to identify users and only allow the person with the matching cookie to see and use the delete buttons.

If you have other solutions or want to claim this one, comment below.

**Comment by MitchWilkins** [comment]
I will try using cookies. I think that isn't a bad fix without implementing a log in system.

**Comment by JacksonBates** [comment]
Yeah, it's a bit of a hack, but it will plug the gap until we do a major upgrade to the backend.
